rampwf/hyperopt/tests/test_hyperparameter.py

```diff
@@ -36,10 +36,25 @@ def test_hyperparameter():
 
 
 @pytest.mark.parametrize("submission", ['starting_kit', 'one_hyper_kit'])
-def test_hyperopt(submission):
+def test_hyperopt_data_label_notNone(submission):
     ramp_kit_dir = os.path.join(
-        PATH, 'interfaces', 'header_in_files', 'titanic')
+        PATH, 'interfaces', 'header_in_files', 'classifier_kit')
+    destination_folder = \
+        os.path.join(ramp_kit_dir, 'submissions',
+                     'one_hyper_kit_titanic_hyperopt')
+    if not os.path.exists(destination_folder):
```

Comment:
why do you need this?

Reply:
for test coverage (to test a if/else branch)